src/unix.rs

```diff
@@ -25,6 +25,13 @@ pub struct Acquired {
     byte: u8,
 }
 
+#[derive(Debug)]
+pub enum ErrFromEnv {
+    ParseEnvVar,
+    OpenFile(String),
```

Comment:
The filename should be `PathBuf` just in case of non-utf8 characters (jobserver currently uses `env::var` so it is guaranteed to be valid utf8, but IMO it should use `env::var_os` instead for non utf-8 filename).

```suggestion
    OpenFile { filename: PathBuf, err: std::io::Error },
```